okhttp/src/main/java/io/grpc/okhttp/OkHttpServerTransport.java

```diff
@@ -489,6 +506,9 @@ public void run() {
           if (keepAliveManager != null) {
             keepAliveManager.onDataReceived();
           }
+          if (maxConnectionIdleManager != null) {
+            maxConnectionIdleManager.onTransportActive();
```

Comment:
This isn't what active means in the context of idle. This is active in the context of keepalive. Active for idle should be places we do `streams.put()` and if the streams was previously empty.